dist-persist/wbstack/src/Settings/ProductionCache.php

```diff
@@ -3,7 +3,7 @@
 // Know about proxies... so that we get the real IP..
 $wgCdnServersNoPurge = [
     # IP range matches current kubernetes pod IPs for GKE
-    '10.8.0.0/14'
+    '10.0.0.0/8'
```

Comment:
Just some thoughts, but maybe this was hardcoded because this was not explicitly configured in terraform? Maybe we could still go with this broad new range (if it even works, not sure)

Another option that comes to my mind is setting these values explicit in terraform and pass them down as environment variables to this config?

(please note, the networking inside gke/k8s is still a pretty blind spot for me, maybe I'm approaching this totally wrong)

Reply:
I was thinking more on alternatives to how we could make this easier to update, possibly (the only way that comes to mind) by using an ENV var instead of continuing to hardcode it, but was thinking maybe there is some good reason I'm missing to why we were hardcoding it in the first place(the wbstack implementation, pre-terraform)? 

If we can't come up with any good reasons I don't see why we should keep hardcoding it and either pass it to the containers as an ENV var from the place that makes the most sense be that terraform or the values file?

Maybe @tarrow has some insights